generators/app/index.js

```diff
@@ -145,6 +159,13 @@ module.exports = class extends BaseGenerator {
         this.skipCheckLengthOfIdentifier = this.configOptions.skipCheckLengthOfIdentifier =
             this.options['skip-check-length-of-identifier'] || this.config.get('skipCheckLengthOfIdentifier');
         this.jhiPrefix = this.configOptions.jhiPrefix = _.camelCase(this.config.get('jhiPrefix') || this.options['jhi-prefix']);
+
+        this.entitySuffix = this.configOptions.entitySuffix =
+            this.config.get('entitySuffix') === false ? '' : this.config.get('entitySuffix') || this.options['entity-suffix'];
```

Comment:
You don't need the conditional, jist short circuiting is enough

Reply:
I need the `this.config.get('entitySuffix') === false` because when the parameters come from the jdl (`entitySuffix false`), it's a real false value. So we need to test it and convert it to an empty string.

If the value stays to `false`, the entity name would be `Foofalse`.

I could have done the test in the `asEntity` function but the separation of concerns principle tells me to not. ;-)

Unless I misunderstood what you mean...

Reply:
Ok understood

Reply:
Not anymore. Deepu's point still stands. The JDL doesn't export the suffixes as false (entity & dto)